docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises.md

```diff
@@ -153,74 +153,68 @@ For mount paths, use a mount point with enough space to contain your node densit
 ![Configure On-Premises Cloud Provider](/images/ee/onprem/configure-onprem-3.png)
 
-## Provision the YugabyteDB nodes
+## Provision YugabyteDB nodes
 
-After finishing the cloud provider configuration, click Manage Instances to provision as many nodes as your application requires.
+After finishing the cloud provider configuration, click **Manage Instances** to provision as many nodes as your application requires.
 
 ### Add nodes
 
 ![Configure On-Premises Cloud Provider](/images/ee/onprem/configure-onprem-4.png)
 
-For each node you want to add, click Add Instances to add a YugabyteDB node. You can use DNS names or IP addresses when adding instances. (Instance ID is an optional user-defined identifier.)
+For each node you want to add, click **Add Instances** to add a YugabyteDB node. You can use DNS names or IP addresses when adding instances (instance ID is an optional user-defined identifier.)
 
 ![Configure On-Premises Cloud Provider](/images/ee/onprem/configure-onprem-5.png)
 
 ### Provision nodes manually
 
-To provision your nodes manually, you have two options:
+To provision your nodes manually, you have the following two options:
 
 * If the SSH user you provided has sudo privileges but requires a password, you can [run the pre-provisioning script](#run-the-pre-provisioning-script).
 
 * If the SSH user doesn't have sudo privileges at all, you need to [set the database nodes up manually](#set-up-database-nodes-manually).
 
-#### Run the pre-provisioning script
+#### How to run the pre-provisioning script
 
-{{< note title="Note" >}}
 This step is only required if you set Manually Provision Nodes to true and the SSH user has sudo privileges which require a password; otherwise, skip this step.
-{{< /note >}}
 
-Follow these steps to manually provision each node using the pre-provisioning Python script. 
+Follow these steps to manually provision each node using the pre-provisioning Python script:
 
 ![Configure On-Premises Cloud Provider](/images/ee/onprem/configure-onprem-6.png)
 
-1. Log into the Platform virtual machine via SSH.
+1. Login to Yugabyte Platform virtual machine via SSH.
 
-1. Access the docker `yugaware` container.
+1. Access the docker `yugaware` container, as follows:
 
     ```sh
-    $ sudo docker exec -it yugaware bash
+    sudo docker exec -it yugaware bash
     ```
 
 1. Copy and paste the Python script prompted in the UI and substitute for a node IP address and mount points. 
-
-    \
-    (Optional) Use the `--ask_password` flag if the sudo user requires password authentication.
-
+Optionally, use the `--ask_password` flag if the sudo user requires password authentication.
+    
     ```output
-    bash-4.4# /opt/yugabyte/yugaware/data/provision/9cf26f3b-4c7c-451a-880d-593f2f76efce/provision_instance.py --ip 10.9.116.65 --mount_points /data --ask_password
+bash-4.4# /opt/yugabyte/yugaware/data/provision/9cf26f3b-4c7c-451a-880d-593f2f76efce/provision_instance.py --ip 10.9.116.65 --mount_points /data --ask_password
```

Comment:
Check preview - looks like this needs to be indented

Reply:
fixed